README.md

```diff
@@ -45,3 +48,19 @@ The tests can now be run with nose:
    
 Note that **you cannot run the sls deploy while the virtualenv is active**
 due to how the serverless Python requirements plugin operates.
+
+## Run Test
+
+1. login to AWS console
+   (mozilla-services use: stage)
+2. Go to Step Functions > Dashboard
+3. Select your state machine
+   (mozilla-services use: "ardere-dev-ardere")
+4. Click on "New Execution" button
+5. Paste your json config into text area
+   (example:  https://github.com/mozilla-services/screenshots-loadtests/ardere.json)
```

Comment:
This 404s and should probably be https://github.com/mozilla-services/screenshots-loadtests/blob/master/ardere.json